target/linux/ath79/dts/qca9563_dlink_dir-842-c.dtsi

```diff
@@ -0,0 +1,152 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	aliases {
```

Comment:
Please move the `aliases` node to the respective boards dts.

It should work the way you have done it, but you are referencing a handle which has to be created in the DT inheriting this dtsi, which might not be obvious on first sight.

Reply:
done, thanks!